src/applications/facility-locator/constants/index.js

```diff
@@ -59,6 +59,7 @@ export const LOCATION_OPTIONS = [
  * Defines the ± change in bounding box size for the map when changing zoom
  */
 export const BOUNDING_RADIUS = 0.75;
+export const EXPANDED_BOUNDING_RADIUS = 1.4;
```

Comment:
Curious:  How did 1.4 get decided upon?  Was that just the minimum bump that worked?

Reply:
Yep.